massgen/tests/test_logger_config.py

```
@@ -71,3 +71,32 @@ def test_console_safe_sink_downgrades_non_utf8_retry_text() -> None:
     assert "[X] Retry (1/3): Choose best answer -> then stop" in written
     assert "[!] Warning" in written
     written.encode("cp1252")
+
+
+@pytest.mark.usefixtures("_isolate_test_logs")
+def test_log_stream_chunk_redacts_secrets() -> None:
+    import massgen.logger_config as logger_config
+
+    openai_key = "sk-proj-testsecret1234567890abcdefghijklmnopqrstuvwxyz"
+    bearer_token = "AIzaSyTestSecret1234567890abcdefghijklmnop"
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to: "bearer_token"

(S105)